### DISCUSSION TITLE
Anura3D in GiD

### DISCUSSION THREAD

**Discussion by Li-Guoyao** [Anura3D/Anura3D_OpenSource#41] [Q&A]
Hello, I'm a postgraduate in geomechanics. I'm interested in MPM of Anura3D and want to simulate pile penetration problems. When I choose problem type with Anura3D in GiD, some errors appear as follows:
Error: repeated name '------------------------__#CB#(0)' in QUESTION field, is equal to field 16 ---------------------__#CB#(0)
Error: repeated name 'material_point_#2__' in QUESTION field, is equal to field 123 material_point_#1__
Error: repeated name 'material_point_#3__' in QUESTION field, is equal to field 123 material_point_#1__
Error: repeated name 'material_point_#6__' in QUESTION field, is equal to field 123 material_point_#1__
Error: repeated name 'material_point_#7__' in QUESTION field, is equal to field 123 material_point_#1__
Error: repeated name 'material_point_#10__' in QUESTION field, is equal to field 123 material_point_#1__
Problem type loaded. 79 conditions read. 1 materials read.
I wonder if it has an impact on subsequent analysis. If so, how to solve it.

**Comment by murphy-john** [comment]
What version of the Anura3D problem type and which version of GiD are you using? Will you check if they are compatible based on the tutorial manual installation instructions?

**Reply by Li-Guoyao** [reply]
Thanks for your reply. I'm using Anura3D2022 and GiD of version 14.0.1. The oedometer test is done without problem, but I'm afraid it might have an impact in future. I downloaded the corresponding GiD as tutorial manual instructed.

**Comment by zhangrf21** [comment]
我也遇到了同样的问题，版本：Anura3D2022 and GiD of version 14.0.1，和教程一致。请问怎么解决？
